### PR TITLE
Translations update from Mattermost Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Mattermost Weblate](https://translate.mattermost.com) for [mattermost-languages-shipped/mattermost-mobile-v2](https://translate.mattermost.com/projects/mattermost/mattermost-mobile-v2/).

Current translation status:

Weblate translation status

### CODE REVIEW

File: assets/base/i18n/en.json [hunk 1]
```diff
@@ -1,976 +1,976 @@
...
-  "video.download_description": "This video must be downloaded to play it.",
-  "video.failed_description": "An error occurred while trying to play the video.",
-  "your.servers": "Your servers"
+    "about.date": "Build Date:",
+    "about.enterpriseEditione1": "Enterprise Edition",
+    "about.enterpriseEditionLearn": "Learn more about Enterprise Edition at ",
+    "about.enterpriseEditionSt": "Modern communication from behind your firewall.",
+    "about.hash": "Build Hash:",
+    "about.hashee": "EE Build Hash:",
+    "about.teamEditionLearn": "Join the Mattermost community a",
```
Comment (Willyfrog) [inline assets/base/i18n/en.json:8]: here is the culprit and it is even wrong, we should not introduce this

**Review by Willyfrog — CHANGES_REQUESTED** [review]
there are changes in the english version which will lead to conflicts, also the change done to that file is a typo

File: assets/base/i18n/en.json [hunk 2]
```diff
@@ -1,976 +1,976 @@
...
-  "user.settings.general.nickname": "Nickname",
-  "user.settings.general.position": "Position",
-  "user.settings.general.username": "Username",
-  "user.settings.notifications.email_threads.description": "Notify me about all replies to threads I'm following",
-  "user.tutorial.long_press": "Long-press on an item to view a user's profile",
-  "video.download": "Download video",
-  "video.download_description": "This video must be downloaded to play it.",
-  "video.failed_description": "An error occurred while trying to play the video.",
-  "your.servers": "Your servers"
+    "about.date": "Build Date:",
```
[the remark this answers is not in the document]
Comment (m1lt0n) [reply assets/base/i18n/en.json:2]: Yes, exactly

[the remark this answers is not in the document]
Comment (m1lt0n) [reply assets/base/i18n/en.json:2]: @Willyfrog I've pushed a commit (from weblate) to fix the only different string. The indentation seems something that weblate did due to the change in the en.json file, if you check the language files, they have an indentation similar to the one of the en file. I think it's good to go now right?

[the remark this answers is not in the document]
Comment (m1lt0n) [reply assets/base/i18n/en.json:2]: Also, checking the settings of the mobile project in weblate, it allowed editing the source file, so I unchecked this setting (as it is unchecked for server etc), so no more editing from translators will be possible.

**Review by m1lt0n — APPROVED** [review]

**Review by Willyfrog — APPROVED** [review]

**Pull request merged**